src/rantanplan/core.py

```diff
@@ -134,7 +1439,7 @@ def get_word_stress(word, pos, tag):
     :return: Dict with [original syllab word, stressed syllab. word, negative index position of stressed syllable or 0 if not stressed]
     :rtype: dict
     """
-    syllable_list = hyphenate(word)
+    syllable_list = syllabify(word)
```

Comment:
Now, this would be `syllable_list, _ = ...` or `syllable_list, alternative_syllable_lists = ...`

Reply:
chose the first one